atomic_reactor/plugins/post_export_operator_manifests.py

```diff
@@ -133,3 +135,27 @@ def run(self):
         shutil.rmtree(manifests_path)
 
         return manifests_zipfile_path
+
+    def _verify_csv(self, manifests_path):
+        """Verify the CSV file from the built image
+
+        :param str manifests_path:
+        :raises: ValueError if more than one CSV files are found, or the single
+            CSV is different from the one in the repo (compared by hash digest).
+        """
+        image_csv = OperatorManifest.from_directory(manifests_path).csv
```

Comment:
this will raise https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/utils/operator.py#L597
Operator bundle may contain only 1 CSV file, but contains more

if in the OUTPUT directory is more CSV files

so maybe catch it and reraise with : in the output operator bundle, are more CSV files ....